Media/CollectionStrategyTree.php

```diff
@@ -72,7 +71,7 @@ public function getCollectionId(
 
         // Create Collection
         return $this->createCollection(
-            $title,
+            'test', // TODO: load title
```

Comment:
My current target was to send a email, after that i will implement the additional features and at last the tab in sulu backend.